logistics.md

```diff
@@ -0,0 +1,144 @@
+{% endcollapse %}
+{% collapse Hackathon mentor %}
+You’ve already attended a hackathon, are keen in welcoming new members in the Brainhack community? As a hackathon mentor, you will make sure that your mentees have followed the participant onboarding, you will spend 1-hour a day with them to help with their on-boarding. 
+{% endcollapse %}
+{% collapse Help desk %}
+Signup for the hackathon helpdesk and answer attendees technical questions. As a member of the helpdesk you will sign up for monitoring the helpdesk chat and answer any question.
+{% endcollapse %}
+{% collapse Social breakout host %}
+Each hub will have a virtual room where people can go and chat, meet with people for their morning or afternoon coffee, lunch, or evening drinks. The social breakout host's role is keep the conversation going in a relaxed and inclusive atmosphere.
+{% endcollapse %}
+{% collapse Social media person %}
+
+{% endcollapse %}
+
+{% endcollapse %}
+
+{% endaccordion %}
+
+#### Platforms
+
+{% accordion software %}
+{% collapse Crowdcast %}
+Each hub will be using Crowdcast for the main sessions, and this content will be streamed on Youtube simultaneously such that those who are not formally registered can follow along as well.
+{% endcollapse %} 
+
+{% collapse Mattermost %}
+* Generic channels: 
+  * [HBM-hackathon](https://mattermost.brainhack.org/brainhack/channels/hbm-hackathon)
+  * [General](https://mattermost.brainhack.org/brainhack/channels/general)
+  * [Town Square](https://mattermost.brainhack.org/brainhack/channels/town-square)
+* Project channel: Each project will have its own channel
+* Help desk: We will have a set of topic and programming language specific channels where you can ping volunteers for help.
+{% endcollapse %}
+
+{% collapse Zoom/Jitsi %}
+Each project will be using Zoom or Jitsi for their individual calls.
+{% endcollapse %} 
+{% endaccordion %}
+
+
+#### HackTrack
+{% accordion HackTrack %} 
+As with previous OHBM hackathons, we've developed a project template that we would like project leads to use for submitting 
+their project proposals.  
+{% collapse Project submission %}
+Each project is allocated to a project helper who will follow the project from submission until the end of the hackathon.
+To make sure that project helpers have enough time to review project submissions, we're setting two deadlines: 
+1. 2 weeks before the hackathon: submit a draft for project helpers to give feedback on
+2. 1 week before hackathon: hard deadline
+
+[**Project checklist**](https://docs.google.com/spreadsheets/d/1SY-Hnhc-bPX0RA47icVukUVVJ-BrX9ykPFQUBY-U1go/edit#gid=0): this is a spreadsheet to help the project helpers making sure that each project is on track and submitted everything.
```

Comment:
Should we change this to a "view only" link? Not that I think that a random person is going to want to come and change our spreadsheet but...

Reply:
yes, definitely!

Reply:
hmm seems like changing it to a view only link will modify permissions for everyone, so we may have to duplicate the link